app/airq/models/clients.py

```diff
@@ -146,12 +146,13 @@ class Client(db.Model):  # type: ignore
         ),
     )
 
-    # Send alerts at most every one hour to avoid spamming people.
+    # Send alerts at most every TWO hours to avoid spamming people.
```

Comment:
LGTM. Can you also update the README to reflect this, in the features section?

Reply:
yeah fixed in seperate PR

Reply:
got it. in the future you can just update this one if that's easier:

* make your change
* `git add -A`
* `git commit <message>`
* `git push`

Reply:
fixed!